cuML/src/glm/ols.h

```diff
@@ -57,33 +40,41 @@ void olsFit(math_t *input, int n_rows, int n_cols, math_t *labels, math_t *coef,
 		math_t *intercept, bool fit_intercept, bool normalize,
 		cublasHandle_t cublas_handle, cusolverDnHandle_t cusolver_handle,
 		int algo = 0) {
+    ///@todo: make this function accept cumlHandle_impl!
```

Comment:
I thought the purpose of this change is to expose `cumlHandle_impl`? Why is this an todo tag and not part of this PR?

Reply:
In order to limit the amount of interface-level changes, to just 2 algos (PCA and tSVD), I dropped a todo note here. Because, if I update this method to accept cumlHandle, then I'd also have to update the full cython file to make sure the CI passes. I'll create another PR to expose interface-level changes for OLS.

Reply:
ah.. you meant this `olsFit` method in the header file! Fair point. I missed this one. Will update this.

Reply:
Done. Thanks for catching this.